packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -211,7 +216,7 @@ async function scanParentDirs(destPath: string, readPackageJson = false) {
     currentDestPath = newDestPath;
   }
 
-  return { hasPackageLockJson, packageJson };
+  return { cliType, packageJson };
```

Comment:
I think we should keep `hasPackageLockJson` around for backwards-compat purposes.

Reply:
The function `scanParentDirs()` is not exported. All the usages are in this file.

What backwards-compat use case are you thinking of?

Reply:
Oh ok, in that case 👍